controllers/external-accounts.js

```diff
@@ -41,4 +44,32 @@ const getExternalAccountData = async (req, res) => {
   }
 };
 
-module.exports = { addExternalAccountData, getExternalAccountData };
+const syncExternalAccountData = async (req, res) => {
+  try {
+    const discordUserData = await getDiscordMembers();
```

Comment:
- can we not just name it `discordMembersData` to keep it consistent with the function name?

Reply:
Members will give us a sense of people with `members` roles soi used this as the variable name.